### ISSUE TITLE
Show long command line with arrow key

### ISSUE THREAD

**Issue by Firesphere** [nicolargo/glances#1553]
#### Description

It would be useful to create an option to wrap the output of the running processes, so the full process becomes visible, including arguments. Running it out of screen is at times not giving enough information

#### Versions

* Glances v3.1.3 with psutil v5.6.3
* Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.3 LTS
Release:	18.04
Codename:	bionic

**Comment by Firesphere** [comment]
I mean that even using the `/`, the full command still often runs out of screen, e.g. parameters passed in. 

e.g.:
`Xorg vt2 -displayfd 3 -auth /run/user/55817/gdm/Xauthority -background none -noreset -keeptty -verbos`

is the maximum displayed. with wrapping, I would be able to see any further arguments passed in to Xorg.

**Comment by nicolargo** [comment]
I understand but the only way to show all command line arguments is to display the command on multiple lines. In this case, if a process has a very long command line, Glances will display only a few processes...

Example with the PyCharm process ==> 16 lines...

```
/home/nicolargo/apps/pycharm-2019.2.3/jbr/bin/java -classpath /home/nicolargo/apps/pycharm
2019.2.3/lib/bootstrap.jar:/home/nicolargo/apps/pycharm-2019.2.3/lib/extensions.jar:/home/nicolarg
/apps/pycharm-2019.2.3/lib/util.jar:/home/nicolargo/apps/pycharm-2019.2.3/lib/jdom.jar
/home/nicolargo/apps/pycharm-2019.2.3/lib/log4j.jar:/home/nicolargo/apps/pycharm-2019.2.
/lib/trove4j.jar:/home/nicolargo/apps/pycharm-2019.2.3/lib/jna.jar -Xms128m -Xmx937m
-XX:ReservedCodeCacheSize=240m -XX:+UseConcMarkSweepGC
-XX:SoftRefLRUPolicyMSPerMB=50 -ea -XX:CICompilerCount=2
-Dsun.io.useCanonPrefixCache=false -Djava.net.preferIPv4Stack=true
-Djdk.http.auth.tunneling.disabledSchemes= -XX:+HeapDumpOnOutOfMemoryError -XX:
OmitStackTraceInFastThrow -Djdk.attach.allowAttachSelf -Dkotlinx.coroutines.debug=off
-Djdk.module.illegalAccess.silent=true -Dawt.useSystemAAFontSettings=lcd
-Dsun.java2d.renderer=sun.java2d.marlin.MarlinRenderingEngine -Dsun.tools.attach.tmp.only=true
-XX:ErrorFile=/home/nicolargo/java_error_in_PYCHARM_%p.log -XX:HeapDumpPath=/hom
/nicolargo/java_error_in_PYCHARM.hprof -Didea.paths.selector=PyCharm2019.2
-Djb.vmOptionsFile=/home/nicolargo/.PyCharm2019.2/config/pycharm64.vmoptions
-Didea.platform.prefix=Python com.intellij.idea.Main
```

**Comment by Firesphere** [comment]
Good point. Maybe a "get full command for currently selected" might be something to consider, but I understand your point. Indeed not really feasible at the moment :)

**Comment by ostrolucky** [comment]
I like how [procs](https://github.com/dalance/procs) is doing it, you can simply hit ← → arrows to scroll through whole args

**Comment by anacrolix** [comment]
> I like how [procs](https://github.com/dalance/procs) is doing it, you can simply hit ← → arrows to scroll through whole args

That was my intuition, to hit right arrow but it didn't do what I expected.

**Comment by wernight** [comment]
+1 on have a shortcut to show the full details including full command line of the selected process. What about using the pinned feature for that?

**Comment by nicolargo** [comment]
Arrow key are already used to switch between sort.

Proposal:
- switch between sort (column) use SHIFT+ARROW
- scroll for long command line: ARROW

**Comment by nicolargo** [comment]
Done on the develop branch

https://github.com/user-attachments/assets/b4e7482a-1d49-4bda-a46b-dffee41e852b
